src/lucky_migrator/create_table_statement.cr

```diff
@@ -1,65 +1,48 @@
 class LuckyMigrator::CreateTableStatement
-  getter statements = [] of String
-  getter rows = [] of String
-  getter table_statement = ""
-  getter index_statements = [] of String
+  private getter rows = [] of String
+  private getter index_statements = [] of String
```

Comment:
Made these private because they're just an implementation detail at this point I think